ci/gpu/build.sh

```diff
@@ -66,7 +66,8 @@ fi
 
 conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
               "rapids-build-env=$MINOR_VERSION.*" \
-              "rapids-notebook-env=$MINOR_VERSION.*"
+              "rapids-notebook-env=$MINOR_VERSION.*" \
+              "python-confluent-kafka"
```

Comment:
Needs a PR into https://github.com/rapidsai/integration as this won't be merged until this line is removed